soh/src/code/z_actor.c

```diff
@@ -662,26 +662,32 @@ s32 Flags_GetSwitch(PlayState* play, s32 flag) {
  * Sets current scene switch flag.
  */
 void Flags_SetSwitch(PlayState* play, s32 flag) {
-    lusprintf(__FILE__, __LINE__, 2, "Switch Flag Set - %#x", flag);
+    u8 changed = !Flags_GetSwitch(play, flag);
```

Comment:
The name `changed` feels weird here, especially as we call `Get` before setting. And some are `Flags_GetX` and some are `!Flags_GetX` (I understand when to use the inversion, just calling out that both cases use `changed`).

Maybe a name like `prevOff`, `previouslyOff`, `wasOff` (and the counter of `prevOn`, etc), would fit better with the intent. Also `willChange` came to mind.